### DISCUSSION TITLE
WanaKana integration

### DISCUSSION THREAD

**Discussion by PvtNoob** [SethClydesdale/genki-study-resources#123] [Ideas]
Hello,
I like your site a lot, but I find Microsoft IME very annoying. Is there a chance that you could implement [WanaKana.js](http://wanakana.com/)? I find it much more comfortable and since I'm from Germany, I keep confusing the Z and Y keys with Microsoft IME, because we use the QWERTZ layout. 

Greetings

**Comment by SethClydesdale** [comment]
Hello @PvtNoob,

I wasn't aware of WanaKana -- it's quite interesting. I think the only downside is that it doesn't look to support Kana --> Kanji conversion. However, I'll consider the idea.

I think the best option would probably be to add it as a new option in the site settings which you can enable, should one not wish to use an IME or doesn't have one installed. It'll take a bit of work, but I'll definitely look into it when I get back to work on it.
